src/content/subgames.cpp

```diff
@@ -141,8 +141,12 @@ SubgameSpec findSubgame(const std::string &id)
 	menuicon_path = getImagePath(
 			game_path + DIR_DELIM + "menu" + DIR_DELIM + "icon.png");
 #endif
+	bool supports_dynamic_shadows = false;
```

Comment:
Can we flip the default to true?
I'm not aware of games opting into waves, waving plants, tonemapping, and other optical features.

Reply:
Games do in fact opt-in waves/waving by declaring in the nodedef which nodes are animated that way. The other examples, sure.

More to the point: I don't know what the other devs think but I think this should be an *opt-in* solution (see also https://github.com/minetest/minetest/pull/11944#issuecomment-1008364488)